### ISSUE TITLE
Sorting issue

### ISSUE THREAD

**Issue by nothingness6** [LykosAI/StabilityMatrix#527]
![StabilityMatrix-02](https://github.com/LykosAI/StabilityMatrix/assets/105210258/4814bd65-b086-417c-8933-af9ad5461f57)
As you can see in the screenshot above, it only shows 1 page when I set the Base Model as all. It works fine when the base model is changed.

**Comment by Vanteal** [comment]
Am having the exact same issue as well.

**Comment by mohnjiles** [comment]
Yeah the civitai API issues are a large part of it. They've changed the way pagination works, removing the page numbers (not sure if bug or intentionally), which is the cause of the primary issue. There are a few other issues on their issue tracker page about it as well: https://github.com/civitai/civitai/issues 

We have an update coming soon that will fix the pagination issues, namely by not using the page numbers and instead using their "next cursor" pointer to the next "page" of results. If the removal of pages wasn't intentional, we'll add those back in a future update.

That said, until they fix other issues with their API, it may still be unstable after our next update.

**Comment by nothingness6** [comment]
> Yeah the civitai API issues are a large part of it. They've changed the way pagination works, removing the page numbers (not sure if bug or intentionally), which is the cause of the primary issue. There are a few other issues on their issue tracker page about it as well: https://github.com/civitai/civitai/issues
> 
> We have an update coming soon that will fix the pagination issues, namely by not using the page numbers and instead using their "next cursor" pointer to the next "page" of results. If the removal of pages wasn't intentional, we'll add those back in a future update.
> 
> That said, until they fix other issues with their API, it may still be unstable after our next update.

Oh, is that mean it's not the issue of SM, but CivitAI API?

**Comment by mohnjiles** [comment]
It looks like the majority of CivitAI API issues were resolved this morning. We should have a fix for our end (to handle the new page stuff) later today.

**Comment by mohnjiles** [comment]
This should be fixed as of the latest [v2.9.2](https://github.com/LykosAI/StabilityMatrix/releases/tag/v2.9.2) - if you encounter any other issues with the model browser, feel free to open another issue. Thanks!

**Comment by nothingness6** [comment]
> This should be fixed as of the latest [v2.9.2](https://github.com/LykosAI/StabilityMatrix/releases/tag/v2.9.2) - if you encounter any other issues with the model browser, feel free to open another issue. Thanks!

I will. Thanks!
